crates/core/component/auction/src/component/dutch_auction.rs

```diff
@@ -184,6 +367,58 @@ pub(crate) trait DutchAuctionManager: StateWrite {
 
 impl<T: StateWrite + ?Sized> DutchAuctionManager for T {}
 
+#[async_trait]
+pub(crate) trait HandleDutchTriggers: StateWrite {
+    /// Process the trigger height for a [`DutchAuction`],
+    async fn process_trigger(&mut self, trigger_height: u64) -> Result<()> {
```

Comment:
should this be `process_triggers` (plural)?

Reply:
It can be, in my mind this was "process a trigger height", but it can be read as "process dutch triggers"